config/config.sample.json

```diff
@@ -0,0 +1,37 @@
+{
+    "ConnectionConfiguration": {
+        "ServerURL": "http://localhost:8065",
+        "WebSocketURL": "ws://localhost:8065",
+        "AdminEmail": "sysadmin@sample.mattermost.com",
```

Comment:
Is this structure given by Mike. I think we a separate struct named adminDetails and use that in connection Configuration

Reply:
this was taken as a reference from an already existing load testing tool of mattermost https://github.com/mattermost/mattermost-load-test-ng/blob/2f31e615fd9ec7c50c95d562048038da9b067457/config/config.sample.json#L2